priv/templates/drab/drab.events.js

```diff
@@ -128,10 +128,8 @@ Drab.setid = function (whom) {
 };
 
 function update_event_handler(node, event, func) {
-  // TODO: learn more about event listeners
-  // node.removeEventListener(event, func)
-  // node.addEventListener(event, func)
-  node["on" + event] = func;
+  node.removeEventListener(event, func);
+  node.addEventListener(event, func);
 }
```

Comment:
Actually, it is not so easy. This is why I commented it out and left it for a future.

This function is called by `Drab.enable_drab_on()` which might be called multiple times. For example, every html changing functions, like `Drab.Live.poke` or `Drab.Element.set_prop` calls it, as it might insert the html partial with nodes with drab attributes.

Simple removeEventListener and addEventListener don't work, as it looks like remove does not delete previous one properly. If you run `Drab.enable_drab_on("body")` you will see that events are triggered multiple times.

Please notice @OvermindDL1's [response on elixirforum](https://elixirforum.com/t/drab-phoenix-library-for-server-side-dom-access/3277/350?u=grych):
*The proper way would be to use addEventListener/removeEventListener with a unique key for the function event, then using the key you can store a name in the data section on the element for later lookup (or stored somewhere else or whatever) so you can cancel it without getting many duplicates.*

TODO: do the test in drab core tests to check if the Drab event is triggered only once.

Reply:
Also, the `on` stuff in jQuery doesn't allow you to clear it as expected either in the face of multiple's, where-as my description that @grych copied will.  :-)